app/react-native/src/preview/components/StoryListView/StoryListView.tsx

```diff
@@ -173,34 +175,38 @@ const StoryListView = ({ selectedStory, storyStore }: Props) => {
     channel.emit(Events.SET_CURRENT_STORY, { storyId });
   };
 
+  const safeStyle = { flex: 1, marginTop: insets.top, marginBottom: insets.bottom + 40 };
```

Comment:
where does this 40 come from?

Reply:
@pvinis I just tried it and I see why you need this 40 now since its the height of the navigation bar, I think we need it but it would be good to make it a constant variable called tabBarHeight for example then it could be `marginBottom: insets.bottom + 40`